babelsberg/cassowary_ext.js

```diff
@@ -26,6 +26,15 @@ ClSimplexSolver.addMethods({
         var constraint = new Constraint(func, this);
         constraint.priority = priority;
         return constraint;
+    },
+    solverName: 'Cassowary',
+    supportsMethods: function() { return false; },
+    supportsSoftConstraints: function() { return true; },
+    supportsFiniteDomains: function() { return false; },
+    supportedDataTypes: function() {
+        // Cassowary does not support strings, but there are actively used scenarios
```

Comment:
"but there are actively used scenarios"... seems about right, we might reword it to "but there are scenarios where the coercion [...] is deliberately used"